apollo-audit/README.md

```diff
@@ -0,0 +1,114 @@
+# Features: Apollo-Audit-Log
+
+This module mainly provides audit log functions for other Apollo modules.
+
+## How to switch 
+
+We can switch this module freely by properties:
+
+by adding properties to application.properties:
+
+```
+# true: enabled the new feature of audit log
+# false/missing: disable it
+apollo.audit.log.enabled = true
+```
+
+## How to use
+
+### Append an AuditLog
+
+We can do this by using annotations:
+
+```java
+@ApolloAuditLog(type=OpType.CREATE,name="App.create")
+public App create() {
+  // ...
+}
+```
+
+Through this, an AuditLog will be created and its AuditScope will be activated during the execution of this method.
+
+Equally, we can use ApolloAuditLogApi to do this manually:
+
+```java
+public App create() {
+  Autocloseable auditScope = api.appendAuditLog(type, name);
+  // ...
+  auditScope.close();
+}
+```
+
+The only thing you need to pay attention to is that you need to close this scope manually～
+
+### Append DataInfluence
+
+This function can also be implemented automatically and manually.
+
+#### Automatically
+
+First, we need to add audit-bean-definition to class of the entity you want to audit:
+
+```java
+@ApolloAuditLogDataInfluenceTable(tableName = "App")
+public class App extends BaseEntity {
+  @ApolloAuditLogDataInfluenceTableField(fieldName = "Name")
+  private String name;
+  private String orgId;
+}
+```
+
+In class App, we define that its data-influence table' name is "App", the field "name" needs to be audited and its audit field name in the table "App" is "Name". The field "orgId" is no need to be audited.
+
+Second, use API's method to append it:
+
+```java
+/**
+ * Append DataInfluences by a list of entities needs to be audited, and their
+ * audit-bean-definition.
+ */
+ApolloAuditLogApi.appendDataInfluences(List<Object> entities, Class<?> beanDefinition);
+```
+
+Actually we don't need to manually call it. We can depend on the DomainEvents that pre-set in BaseEntity:
+
+```java
+@DomainEvents
+public Collection<ApolloAuditLogDataInfluenceEvent> domainEvents() {
+  return Collections.singletonList(new ApolloAuditLogDataInfluenceEvent(this.getClass(), this));
+}
+```
+
+And this will call appendDataInfluences automatically by the listener.
+
+#### Manually
```

Comment:
Maybe should describe how to verify the audit log work too...

For example, which table's data change, or how to check it by UI